src/components/Contacts/ContactDetails/ContactDetailsHeader/ContactDetailsHeader.tsx

```diff
@@ -90,7 +91,7 @@ export const ContactDetailsHeader: React.FC<Props> = ({
       <HeaderBar>
         <ContactAvatar alt={data?.contact.name} src={data?.contact.avatar} />
         <HeaderBarContactWrap>
-          {loading ? (
+          {!data ? (
```

Comment:
You could use the `loading` variable here

Reply:
The benefit of using `!data` instead of `loading` is that type narrowing happens correctly. TypeScript knows that `data` can't be falsy in the other branch, which means that we don't have to use optional chaining unnecessarily.